xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/editors/dataeditors.js

```diff
@@ -753,7 +753,9 @@ editors.XDataEditors = Class.create({
       var movebutton = new Element('span', {
         'class': 'tool move',
         title: 'Drag and drop to change the order'
-      }).update('move');
+      }).update('<span class="presentation">' +
+        $jsontool.serialize($services.icon.renderHTML('arrows')) + 
```

Comment:
I think arrows is the wrong icon here. I don't know if we have a better icon, but this looks way too similar to the cross for removing the field and in some icon themes this is the icon for triggering fullscreen view. We might need to add a new icon for this.

Reply:
There's no icon in silk that would be appropriate for this use case IMO
See https://design.xwiki.org/xwiki/bin/view/Proposal/AddiconstotheXWikiIconSet and https://peacocksoftware.com/silk

That's a bit of a deadlock: to deprecate Silk, we need to replace its uses, but to replace its uses, we would need to have innacurate icons in some places until we deprecate Silk...

Reply:
But couldn't we just use the existing icon for "silk", so basically adding a new icon to silk?

Reply:
The icon we used for this element was a resource which was not part of Silk:
`icons/datamodel/move.png`
It's an extra resource provided in the XWiki web war, in the 'datamodel' folder https://github.com/xwiki/xwiki-platform/blob/bc6fb452ef10414c1756b2b547e8129ec1e35c9d/xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/icons/datamodel/move.png
It looks pretty similar to the silk style, but AFAIK it cannot be mapped using the icontheme...
And it respects its own naming convention, so there's no way to use it in the silk icontheme without a complete overhaul: `move` in datamodel is something different from `move` in silk or FA.

Reply:
We can just copy the icon into the silk folder and then add a mapping in the silk icon theme. See for example #2645 which also adds a new icon to silk. In particular if we have an icon that can be added to silk, this shouldn't hold us back from using proper icons from FA.

Reply:
Changes pushed in https://github.com/xwiki/xwiki-platform/pull/2593/commits/2138cd940c9b04a25e55a15a4fd617f9a88b47ce

Reply:
Note that I'm also pretty sure that we fall back on the default icon theme which currently silk icons when an icon is not found, meaning that even if the user is using an icon theme that doesn't include a mapping for the new icon there is no regression compared to the current state - so this is not an argument for not adding a new icon.

Reply:
I just added the icon `reposition` to the iconTheme (default, silk and FA). I copied the resource from the datamodel folder to the silk one (while renaming it properly). Eventually we'll need to remove it from datamodel, but this resource might still be used in extensions so I don't think we should break this.
I'd prefer to use `move` to name icons for this, and use something along the line of `goto` for the icons we currently have mapped for `move`. However, this is an important breakage of the mapping, and doing this change would increase drastically the scope of this PR. 

See https://github.com/xwiki/xwiki-platform/pull/2593/commits/f3bf12105646ed809d25da90a4a7c7f76d5306e4 for the changes